skyportal/models/classification.py

```diff
@@ -93,7 +93,7 @@ class Classification(Base):
     def to_dict_public(self):
         return {
             'classification': self.classification,
-            'author_name': self.author_name,
+            'author_name': self.author.first_name + ' ' + self.author.last_name,
```

Comment:
Should this fallback to author_name if first and last name aren't defined?

Reply:
It should yes 👍. @antoine-le-calloch if you look at the TNS components of the frontend there is a nice method nice that tries to have firstname + last name (affiliations) and uses fallback when needed. You could do the same. Using the affiliations if available could be a big plus too.